src/Mvc/src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/DefaultModelBindingContext.cs

```diff
@@ -184,6 +188,26 @@ public override ModelBindingResult Result
             }
         }
 
+        private int MaxModelBindingRecursionDepth
+        {
+            get
+            {
+                if (_maxModelBindingRecursionDepth == 0)
```

Comment:
suggestion - use `int?` to make the sentinel value clear

Reply:
:ok: